common-lib/src/main/scala/com/gu/mediaservice/lib/logging/MarkerUtils.scala

```diff
@@ -2,15 +2,28 @@ package com.gu.mediaservice.lib.logging
 
 import net.logstash.logback.marker.LogstashMarker
 import net.logstash.logback.marker.Markers.appendEntries
+import play.api.MarkerContext
+import scala.language.implicitConversions
+
 import scala.collection.JavaConverters._
 
-trait LogMarker { def toLogMarker: LogstashMarker }
+trait LogMarker {
+  def toLogMarker: LogstashMarker
```

Comment:
The implementation of this is the same (other than in `MarkerMap`); is it worth providing a default concrete implementation in the trait which can be overridden if you need to something different?

Reply:
Yeah that's a good idea.